contracts/interfaces/IERC7540.sol

```diff
@@ -0,0 +1,137 @@
+     */
+    function pendingDepositRequest(uint256 requestId, address controller) external view returns (uint256 assets);
+
+    /**
+     * @dev Gets the pending redeem request amount for a given controller and requestId.
+     * @param requestId The unique identifier for the request.
+     * @param controller The address of the controller.
+     * @return shares The amount of shares in the pending state.
+     */
+    function pendingRedeemRequest(uint256 requestId, address controller) external view returns (uint256 shares);
+
+    /**
+     * @dev Gets the claimable deposit request amount for a given controller and requestId.
+     * @param requestId The unique identifier for the request.
+     * @param controller The address of the controller.
+     * @return assets The amount of assets in the claimable state.
+     */
+    function claimableDepositRequest(uint256 requestId, address controller) external view returns (uint256 assets);
+
+    /**
+     * @dev Gets the claimable redeem request amount for a given controller and requestId.
+     * @param requestId The unique identifier for the request.
+     * @param controller The address of the controller.
+     * @return shares The amount of shares in the claimable state.
+     */
+    function claimableRedeemRequest(uint256 requestId, address controller) external view returns (uint256 shares);
+
+    /**
+     * @dev Sets or revokes an operator for the given controller.
+     * @param operator The address of the operator.
+     * @param approved The approval status of the operator.
+     * @return success Whether the operation was successful.
+     */
+    function setOperator(address operator, bool approved) external returns (bool success);
+
+    /**
+     * @dev Checks if an operator is approved for a controller.
+     * @param controller The address of the controller.
+     * @param operator The address of the operator.
+     * @return status Whether the operator is approved.
+     */
+    function isOperator(address controller, address operator) external view returns (bool status);
+
+    /**
+     * @dev Gets the details of a pending deposit request.
+     * @param controller The address of the controller.
+     * @param requestId The unique identifier for the deposit request.
+     * @return request The request object containing amount and claimable details.
+     */
+    function getPendingDepositRequest(
+        address controller,
+        uint256 requestId
+    ) external view returns (Request memory request);
+
+    /**
+     * @dev Gets the details of a pending redeem request.
+     * @param controller The address of the controller.
+     * @param requestId The unique identifier for the redeem request.
+     * @return request The request object containing amount and claimable details.
+     */
+    function getPendingRedeemRequest(
+        address controller,
+        uint256 requestId
+    ) external view returns (Request memory request);
```

Comment:
These two functions are NOT documented in the ERC and are thus not part of the standard. They have nothing to do here.

Reply:
Removed